### ISSUE TITLE
Aram boost doesn't work?

### ISSUE THREAD

**Issue by caerizu** [KebsCS/KBotExt#188]
I reduced my RP to around 320 and it finally said aram boost is eligible so I went into a game and used aram boost and it said it bought Annie and the boost appeared to work only the first time, after the game finished I saw my RP was reduced from boosting and when I tried to boost again it said it was ineligible, I waited an hour and pressed refund RP and nothing even happened, it didn't even show anything in the console thing. I noticed it didn't even buy Annie since I don't see Annie being bought or refunded in the store, does it have something to do with my server since I'm playing on SG?

**Comment by caerizu** [comment]
And I didn't play Annie during that time at all.

**Comment by caerizu** [comment]
Also, it seems like instant message doesn't work either, it auto accepts when a lobby pops up but doesn't send out messages.

**Comment by Kumicoo** [comment]
Yeah idk its kinda weird, got 300 RP, i boost, i refund it like 10 hours after, i try to boost again, it says boost is not available.

I have 300 RP, Singed costs 260 RP.

Idk, @KebsCS ?

**Comment by bit-paper** [comment]
aram boost still normally works. i am playing with it everyday with my friend and no issues util now. you guys doing something wrong. or maybe region, idk. on euw works

**Comment by caerizu** [comment]
> aram boost still normally works. i am playing with it everyday with my friend and no issues util now. you guys doing something wrong. or maybe region, idk. on euw works

Yes, it seems like it's broken in the garena servers cause I used it in OCE and it worked.

**Comment by trinhminhhieu27** [comment]
nah, vn server work fine

**Comment by tragedyxxms** [comment]
> aram boost still normally works. i am playing with it everyday with my friend and no issues util now. you guys doing something wrong. or maybe region, idk. on euw works

Unfortunately, it no longer works. I will describe this problem as best as possible from my side. Unfortunately, but after using the first boost on that day, everything worked normally. I had 638 RP probably, the character it bought was Fizz (-585RP). I played with Hecarim, and after this game the boost no longer works. 

I will wait 24 hours from this message and let you know if anything has changed after that time, maybe it just got bugged and I still "have a boost" or whatever. For this day is the end of boosts from my side, I salute everyone who played with me and got free boosts!
https://shyof-life.is-a.fail/6i9ysNaMw
